src/hotspot/share/runtime/mutex.hpp

```diff
@@ -131,6 +135,7 @@ class Mutex : public CHeapObj<mtSynchronizer> {
   void assert_owner            (Thread* expected)                     NOT_DEBUG_RETURN;
 
  public:
+  static void  add_mutex(Mutex* var);
```

Comment:
This adds a function to a part of the class that previously only held variables. Could this move to a block where we have functions instead?